ESlint controlled via configuration file existence (#63)
deity-io/falcon-docs: docs/falcon-client/configurations.md

### ESLint

> Falcon Client is ESLint ready. To turn it on you need to provide your favorite preset, or you can use ours, [see presets](https://github.com/deity-io/falcon/tree/master/packages/falcon-dev-tools/eslint-config-falcon).

Reviewer: ```suggestion
Falcon Client is ESLint ready. To turn it on you need to provide your favorite preset, or you can [use ours](https://github.com/deity-io/falcon/tree/master/packages/falcon-dev-tools/eslint-config-falcon).
```

> In order to incorporate ESLint you need to create `.eslintrc` file in the Application root. Bellow you can find configuration which uses `@deity/eslint-config-falcon` preset:

Reviewer: ```suggestion
In order to incorporate ESLint you need to create `.eslintrc` file in the Application root. Bellow, you can find a configuration which uses `@deity/eslint-config-falcon` preset:
```

```json
{
  "extends": ["@deity/eslint-config-falcon"],
  "env": {
    "serviceworker": true
  }
}
```

> It is also worth to mention that the file `.eslintignore` needs to be added in order to omit linting `node_modules` or build assets. Bellow, you can find our recommendations:

Reviewer 2: ```suggestion
It is also worth mentioning that the file `.eslintignore` needs to be added in order to omit linting of `node_modules` and built assets. Bellow, you can find our recommendations:
```
